docs/source/app-dev/grpc/index.rst

```diff
@@ -3,25 +3,25 @@
 
 .. _grpc:
 
-gRPC
-####
+Using gRPC with the Ledger API
+##############################
```

Comment:
```suggestion
Using the Ledger API with gRPC
##############################
```

gRPC is the framework used to implement the Ledger API. What developers do is accessing the Ledger API using the tools provided by the gRPC framework (mostly code generation on the client-side). This wording seems more to the point, but feel free to object.